app/javascript/repository.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+<div>
+  <div>{{ repository.name }}</div>
+  <button v-show="!repository.tracked" v-on:click="track()">trackear</button>
+  <button v-show="repository.tracked" v-on:click="untrack()">destrackear</button>
+</div>
+</template>
+
+<script>
+import axios from 'axios';
+
+export default {
+  data: function () {
+    return {
+      repository: this.value,
+    }
+  },
+  props: ['value'],
+  methods: {
+    track() {
+      this.setTrackedStatus(true);
+    },
+    untrack() {
+      this.setTrackedStatus(false);
+    },
+    setTrackedStatus(status) {
+      this.repository.tracked = status;
+
+      axios.put('/api/repositories/' + this.repository.id, { tracked: status })
```

Comment:
Aquí podrías usar `/api/repositories/${this.repository.id}\` con las comillas al revés -> `

Reply:
perfecto!